chains/evm/cli/erc20/deposit.go

```diff
@@ -76,7 +76,10 @@ func ProcessDepositFlags(cmd *cobra.Command, args []string) error {
 		return err
 	}
 	resourceIdBytesArr, err = flags.ProcessResourceID(ResourceID)
-	return err
+	if err != nil {
```

Comment:
I think this `if` is redundant. The behavior will be the same if we just `return err`